### ISSUE TITLE
Fix preview action

### ISSUE THREAD

**Issue by rousik** [flattenthecurve/guide#402]
`translation-updated` workflow has started failing about ~3 days ago. Preview deployments are pushed to vercel but the script fails to determine/find the deployed url and subsequently fails to update PR with the address.

When I compare last successful run and the first failing one I can see that now CLI has been upgraded between the two runs:

This run uses now cli 17.1.1 and succeeds:
https://github.com/flattenthecurve/guide/runs/643046880?check_suite_focus=true

This run uses now cli 19.0.0 and fails (also prints some warnings):
https://github.com/flattenthecurve/guide/runs/681551510?check_suite_focus=true

**Comment by rousik** [comment]
@bcardiff fyi

**Comment by bcardiff** [comment]
Maybe the action https://github.com/amondnet/now-deployment should be replaced with the new one https://github.com/marketplace/actions/vercel-action .

The argument changed for the new zeit/vercel branding.

**Comment by rousik** [comment]
Good idea. PR to do this switch https://github.com/flattenthecurve/guide/pull/403 is awaiting your approval.

**Comment by rousik** [comment]
The action still can't find the deployment and complains that `zeit-team-id` is not known argument. I'm not exactly sure what are the values of these secrets and how vercel and now attributes relate to each other... But the change clearly did not make things worse so at least that's nice.
